msu/hooks/entity/tactical/player.nut

```diff
@@ -30,4 +39,18 @@
 			}
 		}
 	}
+
+	local onSerialize = o.onSerialize;
+	o.onSerialize = function( _out )
+	{
+		this.getFlags().set("LevelUpsSpent", this.m.LevelUpsSpent);
+		onSerialize(_out);
```

Comment:
Also imo it's good practice to remove flags after you serialize with them just to keep everything clean

Reply:
Just to make sure that I understand correctly, you mean that the flag used for serialization should be removed, so that the `tag_collection` doesn't have that entry anymore while the play session is still running? If yes, then I guess that's definitely cleaner. Although, I don't see how leaving the flag in could cause any trouble. Any examples?

Reply:
In this case it shouldn't cause issues, but if you are for example using the existence of a flag to share information, not removing it after serialization will cause issues. Therefore imo it's best to just follow this convention everywhere to make sure you don't run into that issue, as well as saving on a bit of memory :P